src/data-loader.ts

```
@@ -818,12 +842,12 @@
 			const modelsUsed = extractUniqueModels(entries, e => e.model);
 
 			return {
-				sessionId: latestEntry.sessionId,
-				projectPath: latestEntry.projectPath,
+				sessionId: createSessionId(latestEntry.sessionId),
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)